website/docs/d/servicebus_topic_authorization_rule.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Messaging"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_servicebus_topic_authorization_rule"
+description: |-
+  Gets information about a ServiceBus Topic authorization Rule within a ServiceBus Topic.
+---
+
+# Data Source: azurerm_servicebus_topic_authorization_rule
+
+Use this data source to access information about a ServiceBus Topic Authorization Rule within a ServiceBus Topic.
+
+## Example Usage
+
+```hcl
+data "azurerm_servicebus_topic_authorization_rule" "example" {
+  name                = "example-tfex_servicebus_topic"
+  resource_group_name = "example-resources"
+}
+
+output "servicebus_authorization_rule_id" {
+  value = "${data.azurem_servicebus_topic_authorization_rule.example.id}"
+
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - The name of the ServiceBus Topic Authorization Rule resource.
+
+* `resource_group_name` - The name of the resource group in which the ServiceBus Namespace exists.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The ServiceBus Topic ID.
+
+* `namespace_name` - The name of the ServiceBus Namespace.
```

Comment:
`namespace_name` should be in the arguments section as it's required to be supplied?